packages/core/src/infrastructure/database/scripts/setup.ts

```diff
@@ -14,8 +14,19 @@ const __dirname = path.dirname(__filename);
 // This is the full path to the `setup.sql` file.
 const pathToInitFile = path.join(__dirname, 'setup.sql');
 
-exec(`psql -f ${pathToInitFile}`, (_, stdout, stderror) => {
-  if (stderror) {
-    throw new Error(stderror);
+const dbUser = 'postgres';
+const dbName = 'postgres';
+
+const psqlCommand = `psql -U ${dbUser} -d ${dbName} -f ${pathToInitFile}`;
+
+exec(psqlCommand, (error, stdout, stderr) => {
+  console.log(stdout); // Log standard output
+  if (stderr) {
```

Comment:
This was a great catch - we definitely should've only been logging the warnings!